lib/everypolitician/popolo/legislative_period.rb

```diff
@@ -2,6 +2,10 @@ module Everypolitician
   module Popolo
     class LegislativePeriod < Event
       classification 'legislative period'
+
+      def memberships
+        @memberships ||= popolo.memberships.where(legislative_period_id: id)
```

Comment:
I think this needs to also check that it's a membership of the Legislature as well, rather than just any membership of _anything_ that happens to be in this period. At the minute I don't think we have any other types of Memberships, so I'm persuadable that we could defer this until that happens, but I suspect it would be better to be quite strict on this when we know it's going to change RSN.